app/services/post_alerter.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class PostAlerter
+  USER_BATCH_SIZE = 1000
```

Comment:
maybe 100 to keep memory down a tiny bit?

Reply:
👍 Agreed. I took the 1000 default from Activerecord's `find_in_batches`, but our user object it pretty heavy so 100 makes more sense.